dex/templates/service.yaml

```diff
@@ -15,11 +15,12 @@ spec:
   type: {{ .Values.service.type}}
   sessionAffinity: None
   ports:
-{{- range .Values.ports }}
-  - name: {{ .name }} 
-    port: {{ .containerPort }}
-    targetPort: {{ .containerPort}}
-{{- end}}
+  - name: http-grpc
```

Comment:
Please expose the telemetry port as well.

Reply:
thx!